docs/LightningNetwork.md

```diff
@@ -44,122 +45,146 @@ export BTCPAYGEN_LIGHTNING="implementationgoeshere"
 . ./btcpay-setup.sh -i
 ```
 
-For **c-lightning** use `export BTCPAYGEN_LIGHTNING="clightning"`
-
-For **LND** use `export BTCPAYGEN_LIGHTNING="lnd"`
-
-For **eclair** use `export BTCPAYGEN_LIGHTNING="eclair"`
-
-For eclair it also requires `export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-txindex"`
+- For **Core Lightning (CLN)** use `export BTCPAYGEN_LIGHTNING="clightning"`
+- For **LND** use `export BTCPAYGEN_LIGHTNING="lnd"`
+- For **eclair** use `export BTCPAYGEN_LIGHTNING="eclair"` AND `export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-txindex"`
 
 Finally, to begin using Lightning, your blockchain needs to be fully synced.
 
-## Connecting your internal Lightning Node in BTCPay
+# Lightning node configuration in BTCPay
```

Comment:
```suggestion
## Lightning node configuration in BTCPay Server
```


for SEO purposes, please only use H1 once per page (also cuz it's only supposed to be used once =) )